### ISSUE TITLE
v1.1.0-beta35 meshctl demo command fails on post-register checks

### ISSUE THREAD

**Issue by ashleywang1** [solo-io/gloo-mesh#1816]
Here are the logs:
```
✅  Done registering cluster!
🔎 Performing agent post-install checks...
POD LOGS: enterprise-agent-test
Gloo Mesh Registered Cluster Installation
--------------------------------------------

🔴 Gloo Mesh Pods Status
  * deployment "dashboard" has no available pods
  * deployment "prometheus-server" has no available pods
    Hints:
    * check the status of Gloo Mesh deployments with "kubectl -n gloo-mesh get deployments -oyaml"

Agent Configuration
----------------------

🟢 Gloo Mesh CRD Versions

Relay Connectivity
---------------------

🟢 Gloo Mesh Agent Connectivity

{"level":"fatal","ts":"2021-08-17T13:31:04.920Z","caller":"enterprise-agent/main.go:16","msg":"Encountered failed checks.","stacktrace":"main.main\n\t/home/runner/work/gloo-mesh-enterprise/gloo-mesh-enterprise/enterprise-networking/cmd/enterprise-agent/main.go:16\nruntime.main\n\t/opt/hostedtoolcache/go/1.16.3/x64/src/runtime/proc.go:225"}

error: agent post-install check failed: pod enterprise-agent-test failed
```

In the code it's somewhere here: https://github.com/solo-io/gloo-mesh/blob/f3d126ac9066073aac97a89db0fc0bcf3eda8ea6/pkg/meshctl/enterprise/enterprise_utils.go#L312

To reproduce, get meshctl v1.1.0-beta35 and run:
```
meshctl demo istio-multicluster init --enterprise --license $GMG_LICENSE_KEY --version 1.1.0-beta29
```

We should add a bit of a wait before we run the helm test that checks if all the pods are up and running.

**Comment by ashleywang1** [comment]
The output of this check command is actually coming from the `enterprise-agent-test` pod, which runs the `enterprise-agent` image with the `check agent` command.

To fix this, an update/release needs to occur for Gloo Mesh _and_ Gloo Mesh Enterprise.

**Comment by ashleywang1** [comment]
There are 3 PRs involved:
1. k8s-utils: https://github.com/solo-io/k8s-utils/pull/15
2. gloo-mesh https://github.com/solo-io/gloo-mesh/pull/1825
3. gloo-mesh-enterprise https://github.com/solo-io/gloo-mesh-enterprise/pull/1047

**Comment by ashleywang1** [comment]
Tested and fixed in meshctl v1.1.0
